.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+---
+minimum_pre_commit_version: 1.15.2
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.1.0
+    hooks:
+      - id: check-merge-conflict  # Check for files that contain merge conflict strings.
+      - id: trailing-whitespace   # Trims trailing whitespace.
+        args: [--markdown-linebreak-ext=md]
+      - id: mixed-line-ending     # Replaces or checks mixed line ending.
+        args: [--fix=lf]
+      - id: end-of-file-fixer     # Makes sure files end in a newline and only a newline.
+      - id: check-merge-conflict  # Check for files that contain merge conflict strings.
```

Comment:
duplicate `check-merge-conflict` hooks, unless that's intended?

Reply:
Yes, duplicated. Bah!
I'll fix it in another PR since this one is already passing

Reply:
Do the checks really take that long in this repo?

Reply:
Depends on the drone queue.